kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.styled.ts

```diff
@@ -1,5 +1,15 @@
 import styled from 'styled-components';
 
+interface TableCellProps {
+  maxWidth?: string;
+}
+
 export const SwitchWrapper = styled.div`
   padding: 16px;
 `;
+
+export const TableCell = styled.td.attrs({ role: 'tablecell' })<TableCellProps>`
```

Comment:
Is `role: 'tablecell'` intentional? We can query `td`'s with `getAllByRole('cell')`